Sources/Testing/Running/Configuration.TestFilter.swift

```diff
@@ -191,7 +191,7 @@ extension Configuration.TestFilter {
 extension Configuration.TestFilter {
   /// An enumeration which represents filtering logic to be applied to a test
   /// graph.
-  fileprivate enum Operation: Sendable {
```

Comment:
We don't _usually_ bother naming generic parameters more than `T`, `U`, etc.… should we start?

Reply:
I can switch it to `T` for now, I just found this one instance a bit confusing out of context